events/2022.09.Online/TD/Oracle/TDs/WoTWebThing+problemDetails+actionStatus.td.jsonld

```diff
@@ -0,0 +1,386 @@
+			"readOnly": true,
+			"title": "Cycle_Cases_Pressure_Max",
+			"properties": {
+				"Cycle_Cases_Pressure_Max": {
+					"type": "number"
+				}
+			},
+			"forms": [
+				{
+					"href": "http://w3cwotprofile.ddns.net:24042/properties/Cycle_Cases_Pressure_Max",
+					"contentType": "application/json",
+					"op": "readproperty",
+					"additionalResponses": [
+						{
+							"success": false,
+							"schema": "problemDetails"
+						},
+						{
+							"success": true,
+							"schema": "resultValue"
+						}
+					]
+
+				}
+			]
+		},
+		"Cycle_Peak_Operation_Percent_Of_Minute": {
+			"description": "Cycle Peak Operation Percent of Minute",
+			"type": "object",
+			"readOnly": true,
+			"title": "Cycle_Peak_Operation_Percent_Of_Minute",
+			"properties": {
+				"Cycle_Peak_Operation_Percent_Of_Minute": {
+					"type": "number"
+				}
+			},
+			"forms": [
+				{
+					"href": "http://w3cwotprofile.ddns.net:24042/properties/Cycle_Peak_Operation_Percent_Of_Minute",
+					"contentType": "application/json",
+					"op": "readproperty",
+					"additionalResponses": [
+						{
+							"success": false,
+							"schema": "problemDetails"
+						},
+						{
+							"success": true,
+							"schema": "resultValue"
+						}
+					]
+				}
+			]
+		}
+	},
+	"actions": {
+		"power": {
+			"description": "Turns system ON or OFF",
+			"title": "power",
+            "synchronous": true,
+			"input": {
+				"type": "object",
+				"properties": {
+					"value": {
+						"type": "boolean"
+					}
+				}
+			},
+			"forms": [
+				{
+					"href": "http://w3cwotprofile.ddns.net:24042/actions/power",
+					"contentType": "application/json",
+					"op": "invokeaction",
+					"additionalResponses": [
+						{
+							"success": false,
+							"schema": "problemDetails"
+						},
+						{
+							"success": true,
+							"schema": "resultValue"
+						}
+					]
+				}
+			]
+		},
+		"diagnose": {
+			"description": "Runs diagnostics (async action)",
+			"title": "diagnose",
+            "synchronous": false,
+			"forms": [
+				{
+					"href": "http://w3cwotprofile.ddns.net:24042/actions/diagnose",
+					"contentType": "application/json",
+					"op": "invokeaction",
+					"additionalResponses": [
+						{
+							"success": false,
+							"schema": "problemDetails"
+						},
+						{
+							"success": true,
+							"schema": "actionStatus"
+						}
+					]
+				}
+			]
+		},
+		"resetFilter": {
+			"description": "Used to notify the pump that the filter has been replaced. Responds with a 403 Forbidden",
+			"title": "resetFilter",
+			"forms": [
+				{
+					"href": "http://w3cwotprofile.ddns.net:24042/actions/resetFilter",
+					"contentType": "application/json",
+					"op": "invokeaction",
+                    "additionalResponses": [
+						{
+							"success": false,
+							"schema": "problemDetails"
+						},
+						{
+							"success": true,
+							"schema": "resultValue"
+						}
+					]
+				}
+			]
+		}
+	},
+	"events": {
+		"filterClogged": {
```

Comment:
actually, given that this TD uses two profiles, how do I know which one this event follows? Since one profile is baseline and one is webhook, it does not have to follow webhook or does it ?

Reply:
This is just a quesiton, the request for change is the other comment

Reply:
The baseline (basis) profile does not have events at all.
If you need events, you have to use either the SSE or the Webhook Profile.

Reply:
well you have the webhook profile in the top level. I will create a separate issue about this.

Reply:
If you have two profiles listed the TD has to satisfy both.  In this case the baseline profile is a basic set of constraints for HTTP, then the webhook profile adds additional constraints for using webhooks for events.